src/workerd/api/streams/internal-test.c++

```diff
@@ -185,5 +186,72 @@ KJ_TEST("honest small stream") {
   KJ_ASSERT(stream.maxMaxBytesSeen(), 100);
 }
 
+KJ_TEST("WirtableStreamInternalController queue size assertion") {
```

Comment:
Just confirming that without the fix using the `isPiping()` checks in this PR, this test will fail with the queue.size assertion